### ISSUE TITLE
Add styling, update page layout

### ISSUE THREAD

**Issue by j-hannah** [j-hannah/js-report#1]
Improve the very minimal styling now on page.

**Comment by sirkitree** [comment]
Do you have a design in mind, or a site which you think would look similar?

Some similar websites content-wise:
* http://javascriptweekly.com/issues/324
* http://www.ng-newsletter.com/

If we want to try to stick with Lullabot branding, here's our [Lullabot brand guide](https://www.dropbox.com/work/Lullabot%20Design/Design%20Assets/Lullabot%20Brand/brand%20and%20persona%20guide?preview=lullabot_brand-guide.pdf) and of course we could copy some of the general style of lullabot.com such as type styles, link styles.

Here's a quick styleguide I generated from lullabot.com:

![monosnap 2017-03-04 07-04-20](https://cloud.githubusercontent.com/assets/28543/23579345/dceabe92-00a8-11e7-83a6-1a3d31bdfffe.png)
![monosnap 2017-03-04 07-05-02](https://cloud.githubusercontent.com/assets/28543/23579347/ef1d0b7e-00a8-11e7-8cdd-01dcac9be171.png)


TYPOGRAPHY FONT | STYLE | SIZE | LINE HEIGHT | COLOR
---------------------|--------|------|--------------|---------
Header1: FreightSansPro, Helvetica, sans-serif| normal| 16px| 78.4px| #ffffff
Header2: FreightSansPro, Helvetica, sans-serif| normal| 48px| 54px| #0b1b23
Header3: FreightSansPro, Helvetica, sans-serif| normal| 36px| 45px| #2284c0
Body: FreightTextPro, Georgia, serif| normal| 20px| 35px| #0b1b23

**Comment by sirkitree** [comment]
One last thing, the JS logo somewhere would be good too probably.
![images duckduckgo](https://cloud.githubusercontent.com/assets/28543/23579433/4dea019c-00aa-11e7-8adb-6ea4f6bb5c1e.png)

**Comment by sirkitree** [comment]
I'll go with whatever direction you'd like, but happy to work on this a bit. 

Oh, and do we care about it being responsive?

**Comment by j-hannah** [comment]
This is great, @sirkitree! My thoughts on the design were to be minimalist. Mostly to avoid turning it into more work, but also to keep it lightweight. I do think we want it to be responsive (I know the meta tag is missing), although my vision for the page layout is a single column design (at least for now). However, I'm open to changes.

**Comment by sirkitree** [comment]
What do you think about getting it a bit closer to lullabot.com - like our article listing page which is basically single column: https://www.lullabot.com/articles

Here's a list - we don't have to tackle all of them, but maybe we put this in priority order
* [ ] header consistency (JavaScript Report instead of Lullabot)
* [ ] links on the right (just same presentation not a menu)
* [ ] search (is this possible? maybe something typeahead?)
* [ ] intro (try to replicate this a little, not sure how yet)
* [ ] general listing format (byline could be source, use same typo/style)
* [ ] image (probably not have an image)
* [ ] break (maybe we get sponsors at some point?)
* [ ] list signup (do we want to make this an email list?)
* [ ] copyright and follow links in footer

**Comment by j-hannah** [comment]
@sirkitree Good suggestions! Some of this goes beyond what I had in mind for version 1.0. I was thinking uber simple in the beginning - but definitely less ugly than it is right now!  

For example, the header that Lullabot uses is more than I think we'll need initially. I think there will just be 2 pages, so I don't know about a hamburger menu for that.

I definitely want to begin with a mailing list. Also the footer info, follow links, etc. Tomorrow I'm going to add Aphrodite (or at least try). I'd like to try to get some of what you've suggested here into that PR.

Do you have a suggestion for the email newsletter? MailChimp?

**Comment by sirkitree** [comment]
> Some of this goes beyond what I had in mind for version 1.0. I was thinking uber simple in the beginning - but definitely less ugly than it is right now!

Yeah, I don't think we need to do all these things. We can prioritize this list to the ones we'd like to implement now and the rest can do at a later date.

I think the ones that would have the most impact are:
* general listing format
* list signup
* copyright and follow links in footer

Possibly trying to change the header just a bit to make the logo a similar size and width, but that's about it.

> Do you have a suggestion for the email newsletter? MailChimp?

Mailchimp does seem to be the standard these days. We're using it for Lullabot.com (I think) and I'm pretty sure we're using it for Tugboat as well.
